<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TTWatcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/running32.png</normaloff>:/icons/running32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="statLabel">
             <property name="text">
              <string>Statistics</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="LightMaps" name="mapWidget" native="true">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
       </widget>
       <widget class="QCustomPlot" name="graph" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionProcess_TTBIN"/>
    <addaction name="actionDownload_Workouts"/>
    <addaction name="actionShow_in_explorer"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionGo_to_website"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Graph</string>
    </property>
    <addaction name="actionShow_Speed"/>
    <addaction name="actionShow_Heart_Rate"/>
    <addaction name="actionShow_Cadence"/>
    <addaction name="actionShow_Elevation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGraph"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProcess_TTBIN"/>
   <addaction name="actionShow_in_explorer"/>
   <addaction name="actionDownload_Workouts"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProcess_TTBIN">
   <property name="text">
    <string>Process .ttbin...</string>
   </property>
   <property name="toolTip">
    <string>Load a .ttbin file and export to .tcx</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDownload_Workouts">
   <property name="text">
    <string>Download Workouts...</string>
   </property>
  </action>
  <action name="actionShow_in_explorer">
   <property name="text">
    <string>Show in explorer...</string>
   </property>
   <property name="toolTip">
    <string>Show in explorer...</string>
   </property>
  </action>
  <action name="actionShow_Speed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Speed</string>
   </property>
  </action>
  <action name="actionShow_Heart_Rate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Heart Rate</string>
   </property>
  </action>
  <action name="actionShow_Elevation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Elevation</string>
   </property>
  </action>
  <action name="actionShow_Cadence">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Cadence</string>
   </property>
  </action>
  <action name="actionGo_to_website">
   <property name="text">
    <string>Go to website...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LightMaps</class>
   <extends>QWidget</extends>
   <header>lightmaps.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
